package.json

```diff
@@ -19,6 +19,7 @@
   "license": "MIT",
   "repository": "https://github.com/alner/qsSimpleKPI",
   "dependencies": {
+    "natives": "1.1.6",
```

Comment:
not really sure how this is used, does just installing it actually change something, because the examples for it suggests that it needs to be imported for use and if it has to be there, is it a dependency or devdependency?

Reply:
Weird thing was that it solved the problem. It is actually some sort of sub-dependency to babel. And just explicitly adding it as dependency/devDependency seems to force babel to use a newer version. This very much feels like a hack, but nevertheless it works :)

I have moved it to dev dependency though.